packages/server-core/src/seeder.ts

```diff
@@ -7,49 +8,18 @@ import config from './appconfig'
 import { copyDefaultProject, uploadLocalProjectToProvider } from './projects/project/project.class'
 import seederConfig from './seeder-config'
 
-const settingsServiceNames = [
-  'authentication-setting',
-  'aws-setting',
-  'chargebee-setting',
-  'coil-setting',
-  'client-setting',
-  'email-setting',
-  'instance-server-setting',
-  'redis-setting',
-  'server-setting',
-  'task-server-setting'
-]
-
 export async function seeder(app: Application, forceRefresh: boolean, prepareDb: boolean) {
-  if (forceRefresh || prepareDb)
-    for (let config of seederConfig) {
-      if (config.path) {
-        const templates = config.templates
-        const service = app.service(config.path as any)
-        if (templates)
-          for (let template of templates) {
-            let isSeeded
-            if (settingsServiceNames.indexOf(config.path) > -1) {
-              const result = await service.find()
-              isSeeded = result.total > 0
-            } else {
-              const searchTemplate = {}
+  if (!forceRefresh && !prepareDb) return
 
-              const sequelizeModel = service.Model
-              const uniqueField = Object.values(sequelizeModel.rawAttributes).find((value: any) => value.unique) as any
-              if (uniqueField) searchTemplate[uniqueField.fieldName] = template[uniqueField.fieldName]
-              else
-                for (let key of Object.keys(template))
-                  if (typeof template[key] !== 'object') searchTemplate[key] = template[key]
-              const result = await service.find({
-                query: searchTemplate
-              })
-              isSeeded = result.total > 0
-            }
-            if (!isSeeded) await service.create(template)
-          }
-      }
-    }
+  for (const config of seederConfig) {
+    if (!config.path || !config.templates) return
+
+    const templates = config.templates
+
+    const service = app.service(config.path as any) as FeathersService
+
+    await service.create(templates)
```

Comment:
This seems like it removes the check to see if the table has already been seeded (`isSeeded = result.total > 0`) which must run when `prepareDb` is true, as the `Prepare Database` mode is a migration tool for seeding new tables upon updating a deployment. Please add back that logic. We should also be pushing `service.create(templates)` into an array of promises that is then all awaited together with `await Promise.all()` to parallelise and speed up.

Reply:
Update: I have found a way to implement these changes, however I am facing a little problems. Please allow me time till tomorrow 12 pm IST to complete it.

Reply:
I have reintroduced the `prepareDb` changes (checking if the record exists before insert) which were missing. I have also used `Promise.all()` for bulk inserts and single inserts.
Some of _.seed.ts_. files faced problems with bulk inserts, so I added an option `insertSingle` in some _.seed.ts_ files which will insert the templates one by one instead of bulk insert.